### PR TITLE
[SDS-867] Update qtt for python 3.11

### PR DESCRIPTION
**Pull request by bkreynen**
- Update name of requirements lock file
- Update documentation for 3.11 and for WSL
- Update version of qilib

### CODE REVIEW

**Comment by eendebakpt** [comment]
@bkreynen It looks like you need to trigger the build again after the qilib release.

**Comment by bkreynen** [comment]
@eendebakpt the build has passed now and I marked the PR as ready for review.

File: setup.py [hunk 1]
```diff
@@ -37,7 +37,7 @@ def get_package_data(root_dir):
 install_requires = [
     'apscheduler', 'attrs', 'dulwich', 'IPython', 'jupyter', 'lmfit', 'matplotlib',
     'numdifftools', 'numpy>=1.23', 'opencv-python', 'pandas', 'PyQt5', 'pyqtgraph<0.13.0', 'pyvisa', 'pyzmqrpc', 'qcodes>=0.30.0',
```
Comment (eendebakpt) [inline setup.py]: You can unpin pyqtgrapgh. It is not required any more, but it does interfere with other packages (which have pyqtgraph>=0.13)

**Review by eendebakpt — DISMISSED** [review]

File: setup.py (hunk 1 above)
Comment (bkreynen) [reply setup.py]: I unpinned pyqtgraph!

**Review by eendebakpt — APPROVED** [review]

**Pull request merged**